x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -12,11 +12,13 @@ export const UPDATE_RULES_CONFIG_ROUTE_PATH = '/api/csp/update_rules_config';
 
 export const CSP_KUBEBEAT_INDEX_PATTERN = 'logs-cis_kubernetes_benchmark.findings*';
 export const AGENT_LOGS_INDEX_PATTERN = '.logs-cis_kubernetes_benchmark.metadata*';
-export const LATEST_FINDINGS_INDEX_PATTERN = 'cloud_security_posture-findings_latest';
-export const BENCHMARK_SCORE_INDEX_PATTERN = 'cloud_security_posture-benchmark_scores';
+export const LATEST_FINDINGS_INDEX_PATTERN = 'logs-cloud_security_posture.findings_latest-default';
+export const BENCHMARK_SCORE_INDEX_PATTERN = 'logs-cloud_security_posture.scores-default';
 
 export const CSP_FINDINGS_INDEX_NAME = 'findings';
 export const CIS_KUBERNETES_PACKAGE_NAME = 'cis_kubernetes_benchmark';
+export const LATEST_FINDINGS_INDEX_NAME = 'cloud_security_posture.findings_latest';
```

Comment:
Let's not repeat those strings.
Create the patterns from the index names please.

Reply:
done:)